cmd/thanos/compact.go

```diff
@@ -44,6 +44,11 @@ func registerCompact(m map[string]setupFunc, app *kingpin.Application, name stri
 	wait := cmd.Flag("wait", "Do not exit after all compactions have been processed and wait for new work.").
 		Short('w').Bool()
 
+	// TODO(bplotka): Remove this flag once https://github.com/improbable-eng/thanos/issues/297 is fixed.
+	disableDownsampling := cmd.Flag("debug.disable-downsampling", "Disables downsampling. This is not recommended " +
```

Comment:
why is this not `compact.disable-downsampling`? Seems to be the pattern elsewhere in the codebase ... see query.

Reply:
because it's debug only and should be set only when you know consequences.